### DISCUSSION TITLE
Unable to start a new build

### DISCUSSION THREAD

**Discussion by ayman-mph** [codemagic-ci-cd/codemagic-docs#1531] [Q&A]
After selecting the branch to build from and clicking "Start New Build", the  "Start New Build" is disabled and the build is not starting and the pop-up is stuck.
I've tried logging out and logging back in, and also tried to disconnect my GitHub account and connect again, without any results.

**Comment by evelynloo** [comment]
Hi, I am having the same issue too.

**Reply by Ganbabn** [reply]
me too

**Comment by himesh-cm** [comment]
Hi All, it should work ok now, and please let us know if you still are facing any issues with starting builds.
